offer-manager-domain/src/main/groovy/life/qbic/business/customers/update/UpdateCustomerOutput.groovy

```diff
@@ -1,14 +1,20 @@
 package life.qbic.business.customers.update
 
 import life.qbic.business.UseCaseFailure
+import life.qbic.datamodel.dtos.general.Person
 
 /**
- * Output interface for the {@link life.qbic.business.customers.create.CreateCustomer} use case
+ * Output interface for the {@link life.qbic.business.customers.create.UpdateCustomer} use case
  *
  * @since: 1.0.0
- * @author: Tobias Koch
  *
  */
 interface UpdateCustomerOutput extends UseCaseFailure {
+  
+  /**
+   * Is called by the use case, when a customer resource has been updated
+   * @param person The updated person resource
+   */
+  void customerUpdated(Person person)
```

Comment:
Why do you have a Person here but a customer in the other methods?

Reply:
also copied from the other use case, let's talk about it

Reply:
you mean in the daily?

Reply:
we will rename the use cases to use the word "Person" in another PR, thanks for spotting it